client/src/component/Resources/BootcampResourceGet.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import { useSelector } from 'react-redux';
+import Moment from 'react-moment';
+import {Link} from 'react-router-dom';
+
+import {
+  Box,
+  Breadcrumb,
+  BreadcrumbItem,
+  Card,
+  CardHeader,
+  CardHeaderTitle,
+  CardContent,
+  Content,
+  CardFooter,
+  CardFooterItem,
+  Media,
+  MediaContent,
+  Title,
+  Subtitle
+} from 'bloomer';
+
+const BootcampResourceGet = () => {
+  const user = useSelector(state => state.user);
+  const [resources, setResources] = useState([]);
+  const [resourcesFetched, setResourcesFetched] = useState(false);
+
+  const fetchResourcesData = async () => {
+    const resourcesEndpoint = 'http://localhost:3000/resources/getAllResources';
+    const res = await axios.get(resourcesEndpoint);
+    setResources(res.data);
+    setResourcesFetched(!resourcesFetched);
+  };
+
+  useEffect(() => {
+    fetchResourcesData();
+  }, []);
+
+  return (
+    <ResourceWrapper>
+      {resourcesFetched ? (
+        resources.map((resource, i) => {
+          const {
+            id,
+            up_votes,
+            down_votes,
+            title,
+            short_description: descriptionShort,
+            full_description: descriptionFull,
+            resource_url: resourceURL,
+            date_posted: datePosted /* information beyond this line relates to resource poster */,
+            users_id: usersId,
+            email, 
+            first_name: firstName,
+            last_name: lastName,
+            github_url: githubLink,
+            linkedin_url: linkedinLink,
+            bootcamp_name: bootcampAffiliation
+          } = resource;
+          return (
+            <Card key={i} style={{ maxWidth: '600px', margin: '20px' }}>
+              <CardHeader>
+                <CardHeaderTitle>{title}</CardHeaderTitle>
+              </CardHeader>
+              <CardContent>
+                <Media>
+                  <MediaContent hasTextAlign={'left'}>
+                    <Title isSize={5}>Resource Info</Title>
+                    <Subtitle isSize={6}>
+                      <Anchor href={resourceURL}>Link to resource</Anchor>
```

Comment:
Need to make sure all resource_urls taken from users have https:// in front of the URL. Not a problem in this PR since codetrain db has that.